### PR TITLE
chore: change default registry and namespace

### PR DESCRIPTION
**Pull request by JashBook**
1. revert registry and repository
2. add workflow check `default`, `apecloud dockerhub` and `apecloud aliyun` images
ref https://github.com/apecloud/apecloud-cd/pull/389/files
3. release chart to github change all registry to `docker.io` and repository to `apecloud/*`
4. release chart to jihulab change all registry to `apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com` and repository to `apecloud/*`

### CODE REVIEW

File: addons/postgresql/templates/componentdefinition-12.yaml [hunk 1]
```diff
@@ -140,15 +140,15 @@ spec:
     switchover:
       withCandidate:
         exec:
-          image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major12.minor150 }}
+          image: {{ include "redis.image150" . }}
```
Comment (Y-Rookie) [inline addons/postgresql/templates/componentdefinition-12.yaml:143]: why postgresql images named `redis.image150`

File: addons/postgresql/templates/_helpers.tpl [hunk 2]
```diff
@@ -98,4 +98,39 @@ postgresql-15
...
 {{ include "postgresql.componentDefNamePrefix" . }}{{ .Values.componentDefinitionVersion.postgresql15 }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Define image
+*/}}
+{{- define "redis.image140" -}}
```
Comment (Y-Rookie) [inline addons/postgresql/templates/_helpers.tpl:106]: `redis` -> `postgresql`， check them all

**Review by nayutah — APPROVED** [review]

File: addons-cluster/pulsar/values.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ commonAnnotations:
   resource.kubeblocks.io/ignore-constraint: "true"
 
 image:
-  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
+  registry: docker.io
```
Comment (weicao) [inline addons-cluster/pulsar/values.yaml:12]: should also change pulsar addon's cmpv, to make the namespace configurable.

```
      images:
        bookies-recovery: {{.Values.image.registry}}/apecloud/pulsar:2.11.2
```

The namespace is hardcoded to "apecloud"

File: addons/camellia-redis-proxy/values.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@
 image:
   # refer: https://github.com/caojiajun/camellia-jdk21-bootstraps/blob/main/docs/redis-proxy/Dockerfile
   # this is the image of camellia-redis-proxy, which is built from Dockerfile of Apecloud/camellia-redis-proxy forked from caojiajun/camellia-jdk21-bootstraps.
-  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
+  registry: docker.io
   repository: apecloud/camellia-redis-proxy
```
Comment (weicao) [inline addons/camellia-redis-proxy/values.yaml:9]: the namespace is "apecloud"

File: addons/elasticsearch/values.yaml [hunk 5]
```diff
@@ -7,20 +7,20 @@ replicaCount: 1
...
+  repository: elasticsearch
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
   tag: "8.8.2"
   exporter:
-    repository: apecloud/elasticsearch-exporter
+    repository: prometheuscommunity/elasticsearch-exporter
     tag: "v1.7.0"
   plugin:
     repository: apecloud/elasticsearch-plugins
```
Comment (weicao) [inline addons/elasticsearch/values.yaml:19]: namespace is "apecloud"

File: addons/elasticsearch/values.yaml [hunk 6]
```diff
@@ -7,20 +7,20 @@ replicaCount: 1
...
     tag: "8.8.2"
   tools:
     repository: apecloud/curl-jq
```
Comment (weicao) [inline addons/elasticsearch/values.yaml:22]: namespace is "apecloud"

File: addons/camellia-redis-proxy/values.yaml (hunk 4 above)
Comment (JashBook) [reply addons/camellia-redis-proxy/values.yaml:9]: The comments for this image have been added

**Review by Y-Rookie — APPROVED** [review]

**Pull request merged**